stock_demand_estimate/models/date_range.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class DateRange(models.Model):
+    _inherit = "date.range"
+
+    @api.multi
+    @api.depends('date_start', 'date_end')
+    def _compute_days(self):
+        for rec in self:
+            if rec.date_start and rec.date_end:
+                rec.days = (fields.Date.from_string(rec.date_start) -
+                            fields.Date.from_string(rec.date_end)).days + 1
+
```

Comment:
You need to exchange date_start with date_end to have a positive number of days
rec.days = (fields.Date.from_string(rec.date_end) - fields.Date.from_string(rec. date_start)).days + 1